### PR TITLE
Cons/features/tweaks/otp 18199

### PR DESCRIPTION
**Pull request by lisztspace**
Fix a number of smaller issues, add more precision, improve documentation and move tests to common test.

### CODE REVIEW

File: erts/test/erlc_SUITE.erl [hunk 1]
```diff
@@ -435,6 +487,613 @@ make_dep_options(Config) ->
...
+features_macros(Config) when is_list(Config) ->
+    {Run, _, OutDir} = run_fun_dirs(Config),
+
+    Run("f_macros.erl", "", ?OK([])),
+
+    {ok, Peer0, Node0} =
+        ?CT_PEER(#{args => ["-pa", OutDir],
+                   connection => 0}),
+    Call = fun(Node, Fun) ->
+                   rpc:call(Node, f_macros, Fun, [])
```
Comment (bjorng) [inline erts/test/erlc_SUITE.erl:795]: In new code, we should always use `erpc` instead of `rpc`.

File: lib/stdlib/doc/src/erl_features.xml [hunk 2]
```diff
@@ -63,7 +63,20 @@
...
+        <c>rejected</c>) and features that are no longer configurable
+        (status <c>permanent</c>).</p>
+      </desc>
+    </func>
+
+    <func>
+      <name name="configurable" arity="0" since="OTP 25.0"/>
+      <fsummary>Return all configurable features.</fsummary>
+      <desc>
+        <p>Return a list of all configurable features, i.e., features
```
Comment (bjorng) [inline lib/stdlib/doc/src/erl_features.xml:77]:
```suggestion
        <p>Return a list of all configurable features, that is, features
```

(Our documentation guidelines says that we should not use abbreviations such a "i.e." or "e.g.".)

File: lib/stdlib/src/erl_features.erl [hunk 3]
```diff
@@ -187,20 +200,15 @@ keyword_fun(Opts, KeywordFun) ->
                (_) -> false
             end,
     FeatureOps = lists:filter(IsFtr, Opts),
-    {AddFeatures, DelFeatures} = collect_features(FeatureOps),
+    {AddFeatures, DelFeatures, RawFtrs} = collect_features(FeatureOps),
     %% FIXME check that all features are known at this stage so we
     %% don't miss out on reporting any unknown features.
```
Comment (bjorng) [inline lib/stdlib/src/erl_features.erl:205]: Is this FIXME still relevant? If so, could you fix it in this pull request?

**Review by bjorng — CHANGES_REQUESTED** [review]
Overall looks good to me.

File: lib/stdlib/src/erl_features.erl (hunk 3 above)
Comment (lisztspace) [reply lib/stdlib/src/erl_features.erl:205]: No, this has been fixed in this pull request, so I'll just remove the `FIXME`

File: erts/test/erlc_SUITE.erl (hunk 1 above)
Comment (lisztspace) [reply erts/test/erlc_SUITE.erl:795]: Ok, will fix.

**Review by bjorng — APPROVED** [review]

**Pull request merged**